include/rpt/parameters_rpt.h

```diff
@@ -69,13 +69,9 @@ namespace Parameters
     double peak_to_total_ratio;
     double sampling_time; // [s]
 
-    // Parameters to tune or fixed parameters if tuning is disable
-    double dead_time;          // Dead time of the detector per accepted pulse
-    double activity;           // Activity of the tracer
-    double gamma_rays_emitted; // Number of gamma-rays emitted by each
+    // Fixed parameters
+      double gamma_rays_emitted; // Number of gamma-rays emitted by each
```

Comment:
You need to apply automatic indentation. you can run /contrib/utilities/indent_all.sh . Ask Toni, Lucka, Shahab or Olivier if you need help :)!